src/character.cpp

```diff
@@ -3248,7 +3248,7 @@ void Character::normalize()
 void Character::die( Creature *nkiller )
 {
     g->set_critter_died();
-    is_dead = true;
+    set_all_parts_hp_cur( 0 );
```

Comment:
Why this change? If you are doing this, you should probably just remove the variable entirely (I know of no other uses).

Reply:
Cheers, I'll just remove it. It turns out that `Character`s are only considered "dead" when a vital bodypart reaches 0 hp, and that `is_dead` was doing nothing as far as I could tell.